cli/index.js

```diff
@@ -6,6 +6,13 @@ const chalk = require('chalk');
 const version = require('../package').version;
 const {init} = require('./commands/init');
 
+// Make sure we die if we have an unhandled rejection (like a webpack build
+// failure)
+process.on("unhandledRejection", (err) => {
+  console.error(err);
+  process.exit(1);
```

Comment:
```suggestion
  process.exitCode = 1
  throw new Error(`[packer-cli] ${err}`)
```

Reply:
Fixed. I chose to do both log the error and throw because `console.err()` pretty-prints its arguments.